src/StreamManager.cpp

```diff
@@ -301,6 +308,12 @@ bool CInputStreamLibavformat::IsRealTimeStream()
   return m_stream->IsRealTimeStream();
 }
 
+int CInputStreamLibavformat::GetBlockSize() 
+{
+  Log(LOGLEVEL_NOTICE, "Block Size(): %d", m_streamReadChunkSizeKb);
```

Comment:
@AlwinEsch strangely this never get's called and I don't know why. It is logged as a stream property when the inputstream is opened.

Reply:
BlockSize only be used if demuxing in Kodi itself is used and addon not provide if `INPUTSTREAM_SUPPORTS_IDEMUX` set by addon.

Reply:
You can leave away from this addon :smile:

Reply:
Cool